CRM/Utils/Address/BatchUpdate.php

```diff
@@ -28,6 +28,7 @@ class CRM_Utils_Address_BatchUpdate {
   public $geocoding = 1;
   public $parse = 1;
   public $throttle = 0;
+  public $ct = NULL;
```

Comment:
I feel like this might be better expressed as $contactType so its more obvious what the variable is for

Reply:
in update_greeting job , we are using `ct` for `Contact Type`